packages/fxa-settings/src/styles/tailwind.css

```diff
@@ -22,6 +22,10 @@ body {
   @apply absolute -top-20;
 }
 
+.cta-xl {
+  @apply flex-1 font-bold text-base p-4 border-0 max-h-14 rounded-md;
```

Comment:
Since we already have this class in `fxa-content-server` and we just want to share it with settings, we should instead, move it to `fxa-react` and remove it from content-server, so that way it's only defined in one location but shared by both packages.

Reply:
Fixed!